packages/components/atoms/f-button/src/components/_tests/Button.test.js

```diff
@@ -237,6 +280,45 @@ describe('Button', () => {
                     expect(wrapper.attributes('type')).toBeUndefined();
                 });
             });
+
+            // RouterLink type -> incl. `to` attribute
+            describe('when `to` prop is empty :: ', () => {
+                it('should add `type` attribute', () => {
+                    // Arrange
+                    const propsData = {
+                        to: null,
+                        actionType
+                    };
+
+                    // Act
+                    const wrapper = mount(FButton, {
+                        propsData,
+                        stubs: ['router-link']
```

Comment:
You likely don't need the stub in this test I think.

Reply:
I do get an error in the tests. `[Vue warn]: Unknown custom element: <router-link> ... ` 
https://vue-test-utils.vuejs.org/guides/using-with-vue-router.html#using-stubs